src/pages/validation/ValidationPage.tsx

```diff
@@ -281,12 +281,38 @@ export class ValidationPage extends React.Component<ValidationPageProps, Validat
         this.props.setLoading(false);
     }
 
+    isValidScript = () => {
+        if (!this.props.source) {
+            return false;
+        }
+        const scriptToRun = this.props.source.isYamlEditor ? this.state.yamlScript : this.state.visualScript;
+        if (!scriptToRun) {
+            return false;
+        }
+
+        const indexSeparatorInput = scriptToRun.indexOf(":");
+        const input = scriptToRun.substr(1, indexSeparatorInput - 2);
+        const expected = scriptToRun.substring(indexSeparatorInput + 3, scriptToRun.length - 1);
+
+        if (input.trim().length === 0) {
```

Comment:
from line 297 to 303;
```suggestion
        return !(!input.trim() || !expected.trim());
```